docs/getting-started/handling-errors.md

```diff
@@ -147,7 +147,7 @@ app.get('/fruit/:name', async (request, response, next) => {
             return next(new HttpError({
                 statusCode: 400,
                 code: 'INVALID_FRUIT_NAME',
-                message: 'Please provide a fruit name'
+                message: 'A fruit name was not provided'
```

Comment:
`Please provide a fruit name` felt like an instruction to the end user of a GUI, and hypothetically to the app could just take this message value and pass it on for that purpose, but it feels like the error message of an HTTP error should be more of a statement as to the cause of the error?